### DISCUSSION TITLE
Use tags inside a custom Section

### DISCUSSION THREAD

**Discussion by HeadGapped** [pwndoc/pwndoc#472] [Q&A] [answered]
I have made some custom sections/field and placed them inside my Template and can edit them inside my audits. When I try to use tags such as {company.name} and {auditType} inside the text edit box in the sections, it does not populate the information when the audit is downloaded it just shows the tags in place of the company name and audit type.

Can Tags be used inside a custom field Editor component? 

Any help is appreciated.

**Comment by yeln4ts** [comment]
the proper tag to use is {_{company.name}_}

**Reply by HeadGapped** [reply]
I will attach some screenshots as I tried your suggestion, and I was unable to make it work.

Here I have the tags in the custom Field under > Audit Section > Executive Summary (a custom section I created)
![Custom Section Page](https://github.com/pwndoc/pwndoc/assets/135577925/db7f0af0-8d7f-458f-997f-e0f542dfaa4f)

I then create an audit and can see them inside the audit section, I set the company the audit is for in the general section.
![Text editor inside audit](https://github.com/pwndoc/pwndoc/assets/135577925/8269a810-03cf-4510-ab37-7ba251c28de5)

If I now save and download the audit the tags are not filled with any information and just display as is.
![completed audit](https://github.com/pwndoc/pwndoc/assets/135577925/d67f98c2-b59a-4636-a214-4ae26c22a048)

**Comment by yeln4ts** [comment, marked as answer]
Sorry it's a markdown error from github editor
the tag is {\_{company.name}\_}

**Reply by HeadGapped** [reply]
Hey mate, thanks for the response. I was still unable to get it working using the above inside the custom section and just moved the text back into the template to pull in the company name. Unsure what I am doing wrong. Thank you.

**Reply by yeln4ts** [reply]
When you use the correct tag, what is returned in the downloaded report ? The tag directly or nothing ? Can you show screenshots like you did in your previous message ?

**Reply by HeadGapped** [reply]
Hey Yeln4ts, I was able to get it to work when I was reproducing the error for screenshots.

Thanks for the help.
